examples/evm-to-evm-generic-message-transfer/src/contracts/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./sepoliaBase0x4bE595ab5A070663B314970Fc10C049BBA0ad489.js";
```

Comment:
is it possilbe to have better filename for this?

Reply:
ill just name the file `storage.ts`